tests/system/action/poll/test_create.py

```diff
@@ -0,0 +1,104 @@
+from tests.system.action.base import BaseActionTestCase
```

Comment:
We need more tests here, e.g. creating without options, giving global options when it is disabled, giving options with votes which do not fit the pollmethod,...

Reply:
Ok. Please create an issue for this